### ISSUE TITLE
use appropriate media formats and dimensions

### ISSUE THREAD

**Issue by ShynRou** [DistributedCollective/Sovryn-frontend#1495]
Currently many images are embedded in SVGs with inconsistent dimensions and some of them are way to large in file size.

Replace all svg pictures with .jpg or where transparency is required with .png, with reasonable dimensions.

**Comment by tiltom** [comment]
I don't think we should replace all of the `svg` images by `jpg` or `png` images. `SVG` images allow us to reuse them on any resolution (so users can zoom in or use them on retina displays without pixelation). 

Do we really have some performance problems related to this @ShynRou ?

What do you think about it @creed-victor , @pietro-maximoff ?

**Comment by ShynRou** [comment]
@tiltom I'm not speaking about the icons which are actual Vector Graphics just about the ones that embed normal raster images as base64 in the svg, e.g. all the pictures on the Buy SOV page (https://live.sovryn.app/buy-sov).
As for the performance, the Buy SOV page downloads 20MB worth of images, with vastly varying resolutions some to small some way to large. We can also implement multi-resolution with the picture element to support high dpi screens with raster images.

**Comment by tiltom** [comment]
@ShynRou Ok, that makes sense :)

**Comment by pietro-maximoff** [comment]
agree with @tiltom about svg icons, we don't need to replace it, and @ShynRou is right about heavy images (jpg, png), i think we should use for them some plugins, for example - tinypng or maybe convert them into webp format

**Comment by soulBit** [comment]
Migrated to Taiga in https://taiga.sovryn.app/project/sovryn-light/us/164
